package.json

```diff
@@ -24,7 +24,8 @@
         "Peter Matthews (https://github.com/pjdmatts)",
         "John Kwening (https://github.com/jkwening)",
         "Vasanth (https://github.com/Vasanthkesavan)",
-        "Travis Haley (https://github.com/Mallek"
+        "Travis Haley (https://github.com/Mallek",
+        "Eddie Ford (https://github.com/seckboy)"
```

Comment:
Thank you for adding this! You have been - and continue to be a valuable contributor!